frontend/components/product_markup.js

```diff
@@ -0,0 +1,46 @@
+/*
+<div key={ i } className='products-summary'>
+    <div className='products-summary__name'>
+    { datum.name }
+    </div>
+    <div className='products-summary__brand'>
+    <i className='products-summary__brand-by italic'> by </i> { datum.brand }
+    </div>
+    <div className='products-summary__data-input-status'>
+    <i className='italic'> Status </i> { datum.dataInputStatus }
+    </div>
+</div>
+*/
+
+import React, { PureComponent } from 'react';
+
+export default class ProductMarkup extends PureComponent {
```

Comment:
this should be named 'ProductSummary', try to avoid naming things with the name of a data structure or something obvious, it's not a good approach. Also, name things according to what they truly are. Like the word Markup, we know the component renders markup, but a component is more than that, it has other logic, it has other things that it does besides rendering markup. 

We should think about 'what is this component', at a high level its a component that renders a summary of the overall product in the home page, so that seems like a logical name. Same thing with other stuff, i wouldn't call an array type of object 'somethingArray', cuz i know it's an array (unless if it really needs to have that name under some circumstances)